hub/api/tests/test_api_with_compression.py

```diff
@@ -203,3 +203,18 @@ def test_audio(ds: Dataset, compression, audio_paths):
             ds.audio.append(hub.read(path))  # type: ignore
     for i in range(10):
         np.testing.assert_array_equal(ds.audio[i].numpy(), arr)  # type: ignore
+
+
+@enabled_datasets
+@pytest.mark.parametrize("compression", hub.compression.VIDEO_COMPRESSIONS)
+def test_video(ds: Dataset, compression, video_paths):
+    path = video_paths[compression]
+    ds.create_tensor("video", htype="video", sample_compression=compression)
+    sample = hub.read(path)
+    assert len(sample.shape) == 4
```

Comment:
does sample.shape return the num_frames and the HWC? if so, can you assert this comparison instead of len?